docs/guides/pushing-images-to-ECR.md

```diff
@@ -0,0 +1,92 @@
+# Pushing images to ECR
```

Comment:
BTW, I would include both pushing and pulling in this guide. Many people might also use `FROM ...` and the setup is pretty much the same.